src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components/EventCallbackFactory.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.AspNetCore.Components
 {
     public sealed class EventCallbackFactory
     {
+        public EventCallback Create(object receiver, EventCallback callback) => default;
```

Comment:
We need these *identity* functions to create an event callback from an existing callback.

The issue is that we can't really tell the difference in the compiler between you passing a lambda or method group in an attribute vs you passing an existing `EventCallback` value. We have to let overloading figure it out.

Reply:
Is it worth adding code comments about this and the other things you've called out in this PR? If it was worth adding a PR comment to enlighten reviewers, there's a good chance the same need will exist later for code maintainers.

Reply:
The comments are in the source code of the compiler and not in this test code.

It's not valid for someone to look at this file and deduce what changes need to be made without an understanding of what the compiler does and why.

Moreover, I plan to supplant the manual update of these files with a copy-paste of the generated-ref-assembly-code in the future once we have it for the AspNetCore repo, which will leave me without a convenient way to preserve comments.